src/player_hardcoded_effects.cpp

```diff
@@ -622,6 +623,20 @@ void player::hardcoded_effects( effect &it )
                     it.set_duration( 0_turns );
                 }
             }
+            if( one_in( 7200 - ( dur - 360_minutes ) / 4_turns ) ) {
+                add_msg_if_player( m_bad, _( "You are beset with a vision of a prowling beast." ) );
+                for( const tripoint &dest : g->m.points_in_radius( pos(), 6 ) ) {
+                    if( g->m.is_cornerfloor( dest ) ) {
+                        g->m.add_field( dest, fd_tindalos_rift, 3 );
+                        add_msg( m_info, _( "Your surroundings are permeated with a foul scent." ) );
```

Comment:
```suggestion
                        add_msg_if_player( m_info, _( "Your surroundings are permeated with a foul scent." ) );
```
To be in line with the message above.